sphinx/writers/latex.py

```diff
@@ -1833,6 +1840,8 @@ def warner(msg):
             hlcode = hlcode.rstrip()[:-14]  # strip \end{Verbatim}
             self.body.append('\n' + hlcode + '\\end{%sVerbatim}\n' %
                              (self.table and 'Original' or ''))
+            if ids:
+                self.body.append('\\let\\SphinxLiteralBlockLabel\empty\n')
```

Comment:
`\SphinxVerbatimTitle` is reset on `\endVerbatim`. Is there any reason to reset `\SphinxLiteralBlockLabel` explicitly?